ggmolvis/compositor.py

```diff
@@ -29,3 +33,37 @@ def _set_compositor_bg(rgba: tuple[float, float, float, float]):
     links.new(render_layers.outputs['Alpha'], mix_node.inputs[0])
     links.new(render_layers.outputs['Image'], mix_node.inputs[2])
     links.new(mix_node.outputs[0], composite.inputs[0])
+
+
+def _create_frame_image(frame_number: int,
+                        width: int,
+                        height: int,
+                        text: str):
+    img = Image.new('RGBA', (width, height), (0, 0, 0, 0))
+    draw = ImageDraw.Draw(img)
+    font = ImageFont.truetype("/System/Library/Fonts/Geneva.ttf", 80)
```

Comment:
This is obviously not portable. Also, to make matters worse, using the default built-in font from PIL doesn't allow custom font size: https://github.com/python-pillow/Pillow/issues/2695

And, of course, the font size may need to be tuned in some cases unless we scale it "automatically" somehow.